.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     working_directory: ~/TensorComprehensions
     resource_class: xlarge
     docker:
-      - image: tensorcomprehensions/linux-trusty-gcc4.8-tapir5.0-cuda8-cudnn6-py3-conda:x86
+      - image: tensorcomprehensions/linux-trusty-gcc4.8-tapir5.0-cuda8-cudnn6-py3-conda:x86_1
 
```

Comment:
Is there a reason we're not updating the current image instead of making a new one?

Reply:
yep, the current one is stable and is being used by users as well. I will need to sync everything with conda packages, python stuff etc. to be able to switch without hazards. I did see some flakiness earlier with new image that i couldn't repro. other than that, no reason :)